BuildResidentialScheduleFile/measure.rb

```diff
@@ -68,6 +68,11 @@ def arguments(model) # rubocop:disable Lint/UnusedMethodArgument
     arg.setDefaultValue(false)
     args << arg
 
+    arg = OpenStudio::Measure::OSArgument.makeStringArgument('building_id', false)
```

Comment:
New argument for `BuildResidentialScheduleFile`. Required for SFA/MF buildings. Can apply to 1 or all units of the building.